src/log.js

```diff
@@ -144,6 +148,19 @@ class Log extends GSet {
     return Log.findTailHashes(this.values)
   }
 
+  /**
+   * Set a function to be called for every entry in a log that is about to
+   * be joined. The arguments will be the entry to verify and a reference
+   * to the log itself and the verification function should return true
+   * if the entry is valid and false if it is not. Logs containing entries
+   * marked as invalid by this function will not be joined
+   * 
+   * @memberof Log
+   */
+  setCustomVerificationFunction (verifyEntry) {
```

Comment:
What I meant with my previous comment about "no setter" was that we shouldn't have a function/setter at all that the user should/can call to set the verification function AFTER creating the Log instance, but that the only way to use a custom verifier is to pass it through the constructor (at creation time). Adding a new function to do that changes the API and doesn't fit the general idea of the design of the API as "operations on the log" as it changes the behaviour of the instance "dynamically". However, if we pass it in the constructor, it's explicit to the user and usage (ie. the only way to use "custom behaviour" is to pass it at instance creation time). Does that make sense?

If that makes sense, let's remove this function (but keep the documentation, it's awesome! perhaps move it to the constructor docs) as to not introduce it to the API of the log and we're good to merge 👍

Sorry for being picky about this one, I believe it's an important design decision even though it may be a seemingly small issue :) You've done fantastic job on the PR! 👍😃

Reply:
No problem @haadcode, I juts misunderstood your comment about no setter :smile: I can see now that as you said, there are no other functions that make the Log a dynamically changeable structure